wordstats/language_data/t2l/code/cleaning_words.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+import sys
+
+"""
+
+Class that with an aspell dictionary, will clean a freq word list of a language
+from anglicisms or words that doesn't exist
+
+"""
+
+# Freq list file path
+FREQ_WORDS_PATH = ''
+
+# Dictionary file path
+DICTIONARY_PATH = ''
```

Comment:
No, it is added in the main once the user pass by argument the language code.
I can add to the comment that it is modified in main